<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWidget</class>
 <widget class="QWidget" name="ReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="ZoomableLabel" name="labelCoarseImage1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ZoomableLabel" name="labelCoarseImage2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="ZoomableLabel" name="labelFocusImage1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="ZoomableLabel" name="labelFocusImage2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="ZoomableLabel" name="labelFineImage1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ZoomableLabel" name="labelFineImage2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ZoomableLabel" name="labelQualityImage1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="ZoomableLabel" name="labelQualityImage2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnExportPDF">
         <property name="text">
          <string>export pdf</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableLabel</class>
   <extends>QLabel</extends>
   <header location="global">ZoomableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
